gocov/test.go

```diff
@@ -45,3 +146,66 @@ func runTests(args []string) error {
 	}
 	return convertProfiles(coverprofile.Name())
 }
+
+func runAllTests(args []string) (err error) {
+	pkgRoot, _ := os.Getwd()
+	pkg, err2 := relToGOPATH(pkgRoot)
+	if err2 != nil {
+		return err2
+	}
+	// TODO(maruel): This assumes this starts with "./". This is
+	// incorrect,someone could request to run test in a separate package.
+	requestedPath := filepath.Join(pkgRoot, args[0][:len(args[0])-3])
```

Comment:
I think if you just did `go list -e <args[0]>` and split the output by line, you could handle both cases fairly easily.